plinth/actions.py

```diff
@@ -186,8 +186,9 @@ def _run(action, options=None, input=None, async=False, run_as_root=False,
         output, error = proc.communicate(input=input)
         output, error = output.decode(), error.decode()
         if proc.returncode != 0:
-            LOGGER.error('Error executing command - %s, %s, %s', cmd, output,
-                         error)
+            if expecting_error:
+                LOGGER.error('Error executing command - %s, %s, %s', cmd, output,
+                             error)
```

Comment:
Even if the error was expected, it would still create a (misleading) error message...